project/deps.sc

```diff
@@ -5,6 +5,7 @@ object Scala {
   def scala213     = "2.13.12"
   def runnerScala3 = "3.0.2" // the newest version that is compatible with all Scala 3.x versions
   def scala3       = "3.3.1"
+  def scala3Lts    = "3.3" //the full version should be resolved later
```

Comment:
I wonder if we could avoid hardcoding this in the future... perhaps LTS compiler releases could be tagged in the POM smh 🤔 
just thinking out loud, should be ok for now